ivy/functional/frontends/paddle/tensor/math.py

```diff
@@ -412,3 +415,15 @@ def any(x, axis=None, keepdim=False, name=None):
 @to_ivy_arrays_and_back
 def diff(x, n=1, axis=-1, prepend=None, append=None, name=None):
     return ivy.diff(x, n=n, axis=axis, prepend=prepend, append=append)
+
+
+@with_supported_dtypes(
+    {"2.5.1 and below": ("float32", "float64", "int32", "int64")}, "paddle"
+)
+@to_ivy_arrays_and_back
+def increment(x, value=1.0):
+    if x.size == 1:
+        return ivy.inplace_increment(x, value)
+    else:
+        # Call paddle to return the correct exception
+        return paddle.increment(paddle.to_tensor(ivy.to_native(x)), value)
```

Comment:
We are not supposed to import any of the native frameworks (`paddle`, `torch`, `tensorflow` etc.) in our frontends. The frontend functions are only supposed to be implemented as a composition of `ivy` functions only. You'll have to remove this and the `paddle` import up top

Reply:
Yes I can understand, I removed it now. I commented above a bit about why I tried doing this :)